pkg/cloud/aws/services/ec2/bastion.go

```diff
@@ -72,21 +71,21 @@ func (s *Service) ReconcileBastion() error {
 	// TODO(vincepri): check for possible changes between the default spec and the instance.
 
 	instance.DeepCopyInto(&s.scope.ClusterStatus.Bastion)
-	klog.V(2).Info("Reconcile bastion completed successfully")
+	s.scope.V(2).Info("Reconcile bastion completed successfully")
 	return nil
 }
 
 // DeleteBastion deletes the Bastion instance
 func (s *Service) DeleteBastion() error {
 	if s.scope.VPC().IsProvided() {
-		klog.V(4).Info("Skipping bastion deletion in unmanaged mode")
+		s.scope.V(4).Info("Skipping bastion deletion in unmanaged mode")
 		return nil
 	}
 
 	instance, err := s.describeBastionInstance()
 	if err != nil {
 		if awserrors.IsNotFound(err) {
-			klog.V(2).Info("bastion instance does not exist")
+			s.scope.V(2).Info("bastion instance does not exist")
```

Comment:
These bastion logs should probably have a reference to the id or instance name

Reply:
yeah, agreed, will add that

Reply:
sure